cypress/pageobject/Patient/PatientConsultation.ts

```diff
@@ -0,0 +1,95 @@
+export class PatientConsultationPage {
+  selectConsultationStatus(status: string) {
+    cy.get("#consultation_status")
+      .click()
+      .then(() => {
+        cy.get("[role='option']").contains(status).click();
+      });
+  }
+
+  selectSymptoms(symptoms: string) {
+    cy.get("#symptoms")
+      .click()
+      .then(() => {
+        cy.get("[role='option']").contains(symptoms).click();
+      });
+  }
+
+  fillIllnessHistory(history: string) {
+    cy.get("#history_of_present_illness").should("be.visible");
```

Comment:
```suggestion
   cy.get("#history_of_present_illness").scrollIntoView;
    cy.get("#history_of_present_illness").should("be.visible");
```

scrollintoview will help the browser to scroll first to find the element, otherwise the test will fail

Reply:
Done.